tests/e2e/pact/specs/eps-convert.spec.ts

```diff
@@ -35,7 +84,8 @@ jestpact.pactWith(
           uponReceiving: `a request to convert ${desc} message`,
           withRequest: {
             headers: {
-              "Content-Type": "application/fhir+json; fhirVersion=4.0"
+              "Content-Type": "application/fhir+json; fhirVersion=4.0",
+              "Authorization": `Bearer ${process.env.APIGEE_ACCESS_TOKEN}`
```

Comment:
Does this mean an access token will appear in the published pact?

Reply:
Not sure if it's actually a problem, but might raise eyebrows if we start sharing pacts with suppliers

Reply:
hm, good point :thinking: i could revert the changes to the verifier and make a separate verifier for testing auth? would be a bit more work but would avoid this issue